doc/1.manual/x1.htm

```diff
@@ -108,14 +108,12 @@ <H3> <A id=s2> 1.2. other resources </A> </H3>
 <a href="http://www.pure-data.info/"> pure-data.info</a>, which aims to be the
 central resource for Pd, from documentation and
 downloads; to forums, member pages, and a patch exchange.
-You can check <A href=http://puredata.info/docs/BooksAboutPd/>
-puredata.info/docs/BooksAboutPd/</A>.
-
-<P> More documentation (warning: very outdated specially on "how to install Pd") is available on the Pd FLOSS site:
-<A href=https://en.flossmanuals.net/pure-data/_full/>
-en.flossmanuals.net/pure-data/_full/</A> (English) and
-<A href=http://fr.flossmanuals.net/PureData/>
-fr.flossmanuals.net/PureData/</A> (French).
+	
+<P>You can check <A href=http://puredata.info/docs/BooksAboutPd/>
+puredata.info/docs/BooksAboutPd/</A> for more documentation. But please 
+be aware that some references might be very old and outdated. For instance, 
+the FLOSS Manuals deal with installing Pd Extended, which is now an abandoned
+fork of Pd since 2013! Please refer to this manual for more up to date information.
```

Comment:
i think it is generally a bad idea to encode information about the outdatedness of webpages in the offline documentation.
at least where it is possible: just put that info on the https://puredata.info/docs/BooksAboutPd/ *now* and everyone will be receive the info, regardless of which version of Pd they are using (actually the page already says that FLOSS-manuals are outdated)

Reply:
see https://github.com/Lucarda/pure-data/pull/8